src/models/Course.js

```diff
@@ -0,0 +1,77 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class Course extends Model {
+    static associate(models) {
+      Course.hasMany(models.ObjectiveCourse, { foreignKey: 'courseId', as: 'objectiveCourses' });
+      Course.hasMany(models.ActivityReportObjectiveCourse, { foreignKey: 'courseId', as: 'activityReportObjectiveCourses' });
+
+      Course.belongsToMany(models.ActivityReportObjective, {
+        through: models.ActivityReportObjectiveCourse,
+        foreignKey: 'courseId',
+        otherKey: 'activityReportObjectiveId',
+        as: 'reportObjectives',
+      });
```

Comment:
please move this to activityReportObjectiveCourse.js

Reply:
Not sure why I would move this it models the same config as ActivityReportObjectiveFile.js?

Reply:
So File <> ActivityReportObjectiveFile

Reply:
done.